edgexfoundry/edgex-ui-go
# refactor: Update to v3 modules and API

closes #597







If your build fails due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-ui-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  <link to docs PR>

## Testing Instructions
Run `make docker`
From compose builder run `make run no-secty ui-dev ds-virtual ds-rest`
Verify UI is operational
- See both device services
- See all devices
- See all device profiles
- Can add/delete/update device for Device Rest

From compose builder run `make clean`
From compose builder run `make run ui-dev ds-virtual ds-rest`
From compose builder run 'make get-token'
Verify UI login with token
Verify UI is operational
- See both device services
- See all devices
- See all device profiles
- Can add/delete/update device for Device Rest

## New Dependency Instructions (If applicable)
